numpy/core/records.py

```diff
@@ -606,6 +628,17 @@ def fromarrays(arrayList, dtype=None, shape=None, formats=None,
     >>> x1[1]=34
     >>> r.a
     array([1, 2, 3, 4])
+    
+    >>> x1 = np.array([1,2,3,4])
+    >>> x2 = np.array(['a','dd','xyz','12'])
+    >>> x3 = np.array([1.1,2,3,4])
+    >>> r = np.core.records.fromarrays(
+            [x1, x2, x3], names='d,e,f',
+            dtype=np.dtype([('a', np.int32), ('b', 'S3'), ('c',np.float32)]))
+    >>> r # names is ignored in favor of dtype
```

Comment:
```suggestion
    >>> r = np.core.records.fromarrays(
            [x1, x2, x3],
            dtype=np.dtype([('a', np.int32), ('b', 'S3'), ('c',np.float32)]))
    >>> r
```
The example above this one demonstrates the `names` kwarg. I think there should be at least one positive example of how to specify the dtype explicitly, without the extra confusion from trying to illustrate how the kwargs are resolved. If you want to add an example that demonstrates how the additional kwargs are ignored when a dtype is explicitly specified, I think it would be best to add it in an additional example below this one (though I don't think it's necessary - the description in the Parameters section is quite clear on this issue).